arm-web/2015-08-01/swagger/AppServiceCertificateOrders.json

```diff
@@ -821,47 +821,36 @@
     "AppServiceCertificate": {
```

Comment:
We generate the swagger from our REST API. The model AppServiceCertificate was used as a top-level resource and as a property in another resource. In swagger then the definition AppServiceCertificate had the ARM envelope properties (id,name,kind,location, sku etc.) even when a property of another resource. This was an issue as it did not represent the actual payload. The change here now has 2 models, AppServiceCertificate and AppServiceCertificateResource and both are used.